Analytics/Classes/SEGAnalytics.m

```diff
@@ -111,6 +111,8 @@ - (void)handleAppStateNotification:(NSNotification *)note
         [self _applicationDidFinishLaunchingWithOptions:note.userInfo];
     } else if ([note.name isEqualToString:UIApplicationWillEnterForegroundNotification]) {
         [self _applicationWillEnterForeground];
+    } else if ([note.name isEqualToString: UIApplicationDidEnterBackgroundNotification]) {
```

Comment:
I believe this would be fired even if the application is being suspended, which is not the behavior that is desired in the PRD.

However, I don't believe it's possible to accurately capture the behavior documented in the PRD, and this code as implemented is the right solution, and we should update the PRD instead.

In case the PRD is not updated, then we might have to update the code here.

Reply:
Agreed to update the PRD - so this part is 👍

Reply:
Fwiw, you're going to get applicationDidEnterBackground when the device is powered off and the app is still active, as well as if they force-quit from the task switcher.  If we're cool with that, then approved.